results.go

```diff
@@ -99,9 +99,13 @@ func (r ErrorResult) Apply(req *Request, resp *Response) {
 	// net/http panics if we write to a hijacked connection
 	if req.Method != "WS" {
 		resp.WriteHeader(status, contentType)
-		b.WriteTo(resp.Out)
+		if _, err := b.WriteTo(resp.Out); err != nil {
```

Comment:
I think, we can handle gracefully errors in `results.go` and log it errors instead of `panic` all over. @revel/core thoughts?